rom-hacking/scripting/animation-commands/ac-dir-val.md

```diff
@@ -0,0 +1,42 @@
+# (778) _AC_DIR_VAL
+
+## Effect
+
+Turns the actor of the animation to a degree angle over a given amount of frames.
+
+- 0°, 360°, -360° = down
```

Comment:
Putting this in an info blurb under the rest of the description might look better.